lib/rules/disallow-space-after-keywords.js

```diff
@@ -51,7 +51,9 @@ module.exports.prototype = {
         );
 
         if (keywords === true) {
-            keywords = defaultKeywords;
+            keywords = defaultKeywords.filter(function(keyword) {
+                return ['function', 'return'].indexOf(keyword) === -1;
```

Comment:
Oops actually I mean't to remove this after changing `nextToken.type !== 'Punctuator'`